setup.py

```diff
@@ -70,4 +94,12 @@ def _export_version(version, sha):
     # Package Info
     packages=find_packages(exclude=["test*", "examples*"]),
     zip_safe=False,
+    extras_require={
+        "scipy": ["scipy"],
+    },
```

Comment:
Do we need `scipy` here?

Reply:
I removed this dependency. It was copied from S3 Plugin initially, and I don't think it's required in S3 IO.